stream_zip.py

```diff
@@ -358,7 +359,7 @@ def _no_compression_32_local_header_and_data(name_encoded, mod_at_encoded, exter
 
         def _no_compression_buffered_data_size_crc_32(chunks, maximum_size):
             # We cannot have a data descriptor, and so have to be able to determine the total
-            # length and CRC32 before output ofchunks to client code
+            # length and CRC32 before output of chunks to client code
```

Comment:
I'm being slightly nit-picky, but can this be in another commit (or even, another PR that we can merge in quickly)? To make the history just that tiny bit clearer.